### ISSUE TITLE
[beta] Night mode, buttons do not have a "web safe" border

### ISSUE THREAD

**Issue by tommygebru** [freeCodeCamp/freeCodeCamp#12836]
Working on MBP in Chrome browser

The page in night mode:
http://beta.freecodecamp.com/en/settings

 has 3 unselected buttons with a very dark border, is not accessible or "colorsafe"
.btn-twitter
.btn-linkedin
.btn-github
 
these buttons borders should be changed to
`border:inherit;`
```


i have provided a screenshot in another comment
to show the difference, linkedin does not have a proper border-color

**Comment by tommygebru** [comment]
![screen shot 2017-01-25 at 7 29 37 am](https://cloud.githubusercontent.com/assets/7606025/22296984/bfef5e92-e2d0-11e6-92c2-4a625eaae133.png)

**Comment by dhcodes** [comment]
@tommygebru we recently made the off-on buttons have a border-color of `#777` on staging if you want to see if that solves it for the off-on switches. @raisedadead want to update the social ones too?

**Comment by tommygebru** [comment]
Very cool #777 should be fine, a little brighter wouldnt hurt either

**Comment by dhcodes** [comment]
Closing as resolved.
